### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by saifudinbadani**
Screen Shot 2022-02-10 at 6 35 00 pm
Screen Shot 2022-02-10 at 6 34 52 pm

### CODE REVIEW

**Review by kotesh-arya — APPROVED** [review]
Well crafted, seems to be good enough for a merge 👍🏻

**Review by ark2002 — APPROVED** [review]
good  to go

File: components/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import "../../shared.css";
+
+.spaced-list,
+.stacked-list{
+    border:1px solid black;
```
Comment (SRUJANA-PENUGONDA13) [inline components/lists/lists.css:5]: you can use var for color black

File: components/lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    width: 15rem;
+    background-color: var(--body);
+    border-radius: 5px;
+}
+
+.spaced-list > .list-heading{
+    border-bottom: 1px solid black;
```
Comment (SRUJANA-PENUGONDA13) [inline components/lists/lists.css:12]: same here

File: components/lists/lists.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+.list-item{
+    font-size: 1.2rem;
+    text-align: center;
+    cursor: pointer;
+}
+
+.list-item:hover{
+    background-color: var(--primary);
+    color: white;
```
Comment (SRUJANA-PENUGONDA13) [inline components/lists/lists.css:29]: same here

**Comment by SRUJANA-PENUGONDA13** [comment]
Everything looks good. Added few suggestions once look into it.

**Pull request merged**